### PR TITLE
Amd subtools

### PR DESCRIPTION
**Pull request by ZheHuang96**
Hi Rikard,

According to the discussion today, I changed some options wich related to` run_amd` function and make this PR. About the separator problem, I still need to fix some problem for the integration test. I will re-looked at it tomorrow.

Best regards,
Zhe Huang

### CODE REVIEW

**Comment by ZheHuang96** [comment]
Hi Rikard, 
The tox check failed when test `transform_etas_boxcox` with eta commit In this case, we have a `eta(1)` and I guess we need to keep it as uppercase?

File: tests/modeling/test_modeling.py [hunk 1]
```diff
@@ -1937,11 +1937,6 @@ def test_zo_abs_on_nl_elim(testdata):
             'ETAB2 = (EXP(ETA(2))**THETA(5) - 1)/THETA(5)',
             'CL = TVCL*EXP(ETAB1)\nV = TVV*EXP(ETAB2)',
         ),
-        (
```
Comment (rikardn) [inline tests/modeling/test_modeling.py:1940]: Couldn't this stay with 'ETA(1)' instead of 'eta(1)' ?

Comment (rikardn) [reply tests/modeling/test_modeling.py:1940]: True, perhaps a too small difference. You don't have to change it.

**Review by rikardn — APPROVED** [review]
I think this looks good.

File: tests/modeling/test_modeling.py (hunk 1 above)
Comment (ZheHuang96) [reply tests/modeling/test_modeling.py:1940]: Ok, thank you!

**Comment by ZheHuang96** [comment]
Thank you so much:)

**Pull request merged**